gammapy/estimators/map/excess.py

```diff
@@ -195,42 +195,63 @@ def run(self, dataset):
         result = self.estimate_excess_map(resampled_dataset)
         return result
 
-    def estimate_excess_map(self, dataset):
-        """Estimate excess and ts maps for single dataset.
-
-        If exposure is defined, a flux map is also computed.
+    def estimate_kernel(self, dataset):
+        """Get the convolution kernel for the input dataset.
 
         Parameters
         ----------
-        dataset : `MapDataset`
-            Map dataset
-        """
+        dataset : `~gammapy.datasets.MapDataset`
+            Input dataset.
 
+        Returns
+        -------
+        kernel : `~astropy.convolution.Tophat2DKernel`
+            Kernel
+        """
         pixel_size = np.mean(np.abs(dataset.counts.geom.wcs.wcs.cdelt))
         size = self.correlation_radius.deg / pixel_size
         kernel = Tophat2DKernel(size)
+        return kernel
```

Comment:
For consistency with the `TSMapEstimator` I think we should return a `Map` object here.